### PR TITLE
[EBS-21] UX Improvements

### PR DESCRIPTION
**Pull request by SteveWestbrook**
Little UX fixes as described below:

-Hide ‘frame dropping’ indicator. (shows when stream is connected)
- Change the FPS section of EBS to say the following: Frame Rate Quality (Remove the dropdown for FPS).
https://gyazo.com/8eb4a1cd02610cf92eda36f75da5882d
-In the FPS dropdown, instead of it showing all sorts of FPS options, make the dropdown show two.
```
Maximum (Recommended) *( 60fps)
Medium (CPU Saver) * )( 30fps)
Low (Max CPU Savings) ( 24fps)
```
Remove all other fps - can be shown by selecting "Show Advanced Options." 
- Make it so when EBS is installed, all audio devices are disabled by default. Right now, when you install EBS or Mac or PC, the Mic/Aux audio is enabled. Please have it disabled by default. All audio options should be disabled by default https://gyazo.com/f39af464355e9b18e6712b9461bc50f9
- Have ‘Desktop Audio’ disabled by default for PC. I don’t think Mac has this. 
- Hide ‘Studio Mode.’ https://gyazo.com/bcb30289ae4acd8f51e3fa6b8e5cad46
- Hide Rec at the bottom of EBS. https://gyazo.com/0788507922df61715994df61521f23df (this was already done)
- Have default fps be 60 and downscale filter default to bilinear. https://gyazo.com/d9e436fc1d11f9825c8dedb100c3243a
- Hide recording, stream delay and auto reconnect in advanced settings.
https://gyazo.com/10d0dbe4be5186e1d339a0ed9f7518d8
- Why not hide all of Advanced settings unless hotkey down when opening settings
- Is it possible that when you add a source, it ‘transforms and fits to screen’ by default. That way, no matter what video source you select, it’s always full screen on the EBS canvas by default. Right now, users add the video source. Then, right click on the image in EBS, go to transform, then fit to screen. 
- Hide several things in the output tab in EBS settings https://gyazo.com/da9157aacffdd2bd4f3c5d2551a1f2b4
- The only thing in help should be log files https://gyazo.com/70ecfcd1fbef8d5af5472b92eb512afe
- Hide ‘auto config wizard’ in tools https://gyazo.com/0b60ac2eb4d79dc1eb3b66f4a54fafae

### CODE REVIEW

File: vendor_skins/Evercast/UI/data/locale/en-US.ini [hunk 1]
```diff
@@ -529,6 +529,9 @@ Basic.MainMenu.Help.Logs="&Log Files"
 Basic.MainMenu.Help.Logs.ShowLogs="&Show Log Files"
 Basic.MainMenu.Help.Logs.UploadCurrentLog="Upload &Current Log File"
 Basic.MainMenu.Help.Logs.UploadLastLog="Upload &Last Log File"
+Basic.MainMenu.Help.CrashLogs="&Crash Logs"
+Basic.MainMenu.Help.CrashLogs.ShowLogs="&Show Log Files"
```
Comment (chadfurman) [inline vendor_skins/Evercast/UI/data/locale/en-US.ini:533]: What's the & for?  Reminds me of pass-by-reference

File: vendor_skins/Evercast/UI/data/locale/en-US.ini [hunk 2]
```diff
@@ -694,6 +698,7 @@ Basic.Settings.Video.DownscaleFilter.Lanczos="Lanczos (Sharpened scaling, 32 sam
 
 # basic mode 'audio' settings
 Basic.Settings.Audio="Audio"
+Basic.Settings.Audio.Disabled="Disabled"
```
Comment (chadfurman) [inline vendor_skins/Evercast/UI/data/locale/en-US.ini:701]: So many little details!  Good find

File: vendor_skins/Evercast/UI/forms/OBSBasicSettings.ui [hunk 3]
```diff
@@ -224,7 +224,7 @@
               <x>0</x>
               <y>0</y>
               <width>753</width>
-              <height>920</height>
+              <height>943</height>
```
Comment (chadfurman) [inline vendor_skins/Evercast/UI/forms/OBSBasicSettings.ui:227]: Wow that's odd... what's the rationale here?  Just curioous :)

File: vendor_skins/Evercast/UI/window-basic-main.cpp [hunk 4]
```diff
@@ -688,21 +688,9 @@ static inline bool HasAudioDevices(const char *source_id)
...
-		ResetAudioDevice(App()->OutputAudioSource(), "default",
-				 Str("Basic.DesktopDevice1"), 1);
-	if (hasInputAudio)
-		ResetAudioDevice(App()->InputAudioSource(), "default",
-				 Str("Basic.AuxDevice1"), 3);
-}
-
 void OBSBasic::CreateDefaultScene(bool firstStart)
 {
+	UNUSED_PARAMETER(firstStart);
```
Comment (chadfurman) [inline vendor_skins/Evercast/UI/window-basic-main.cpp:693]: This is an amazing function name.

File: vendor_skins/Evercast/UI/window-basic-settings.cpp [hunk 5]
```diff
@@ -835,6 +835,41 @@ OBSBasicSettings::~OBSBasicSettings()
 	EnableThreadedMessageBoxes(false);
 }
 
+static inline std::string MapFPS(QString val)
```
Comment (chadfurman) [inline vendor_skins/Evercast/UI/window-basic-settings.cpp:838]: How did you find all this?

File: vendor_skins/Evercast/UI/window-basic-settings.cpp [hunk 6]
```diff
@@ -1460,14 +1514,29 @@ void OBSBasicSettings::LoadResolutionLists()
 	ui->outputResolution->lineEdit()->setText(outputResString.c_str());
 }
 
+static inline void AssignFPSCommonValues(OBSBasic *main, Ui::OBSBasicSettings *ui)
```
Comment (chadfurman) [inline vendor_skins/Evercast/UI/window-basic-settings.cpp:1517]: Hmm... not react that's for sure :)

**Review by chadfurman — APPROVED** [review]

File: vendor_skins/Evercast/UI/data/locale/en-US.ini (hunk 1 above)
Comment (SteveWestbrook) [reply vendor_skins/Evercast/UI/data/locale/en-US.ini:533]: Actually that's a Windows thing - The letter following the ampersand will be underlined, and can be used as a keyboard shortcut, e.g. Alt-F Alt-A will generally take you to (F)ile -> Save (A)s

File: vendor_skins/Evercast/UI/window-basic-settings.cpp (hunk 5 above)
Comment (SteveWestbrook) [reply vendor_skins/Evercast/UI/window-basic-settings.cpp:838]: Oh, this was just a requirement - there were existing values which I wanted to map to the new ones Brad defined.

File: vendor_skins/Evercast/UI/data/locale/en-US.ini (hunk 1 above)
Comment (chadfurman) [reply vendor_skins/Evercast/UI/data/locale/en-US.ini:533]: Ah I remember this from VB now that you mention it!

Comment (SteveWestbrook) [reply vendor_skins/Evercast/UI/data/locale/en-US.ini:533]: That's where I first learned it too - VB3, IIRC :D

**Pull request merged**